### PR TITLE
feat(payment&&orders):#187300121

### PR DESCRIPTION
**Pull request by Mukunzijames**
What does this PR do?

paying with credit card and debit card by using stripe  and buyer || seller can be able to view their orders

Description of Task to be completed?
*paying with credit card and debit card by using stripe  and buyer || seller can be able to view their orders

A user should be able to see all available orders

What are the relevant pivotal tracker/Trello stories?

[#187300121]

### CODE REVIEW

**Review by RWEMAREMY — APPROVED** [review]
good

**Review by musabehonore — APPROVED** [review]

File: src/app/(cart)/dashboard/cart/page.tsx [hunk 1]
```diff
@@ -38,6 +37,25 @@ const Cart: React.FC = () => {
...
+            Authorization: `Bearer ${localStorage.getItem('token')?.replaceAll('"', '')}`
+          }
+        }
+      )) as any;
+
+      if (!response.ok) {
+        throw new Error('Payment failed');
+      }
+      const data = await response.json();
+      window.location.href = data.Success_url;
```
Comment (fab-ryan) [inline src/app/(cart)/dashboard/cart/page.tsx]: please use redux

File: src/app/dashboard/orders/page.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    dispatch(setLoading(true));
+    axiosRequest('GET', '/product-orders', null, true)
+      .then(res => {
+        dispatch(setorders(res.data.data));
+        dispatch(setLoading(false));
+        console.log(res.data.data);
+      })
+      .catch(err => {
+        dispatch(setError(err.message));
+      });
```
Comment (fab-ryan) [inline src/app/dashboard/orders/page.tsx]: here also

**Review by fab-ryan — CHANGES_REQUESTED** [review]
and show me how a seller can view his orders

**Review by fab-ryan — APPROVED** [review]

**Pull request merged**